how-to/customize-workspace/client/src/modules/init-options/interop/init-options.ts

```diff
@@ -1,4 +1,5 @@
 import type { Logger, LoggerCreator } from "../../../logger-shapes";
+import type { ModuleDefinition } from "../../../module-shapes";
```

Comment:
why did we rename it to init options if the module type is initi options in the folder name? This one handles actions? and there can be more registered (e.g. the share one)

Reply:
Reverted